### ISSUE TITLE
Tag input hidden on Qt5 build

### ISSUE THREAD

**Issue by dae** [ankitects/anki#1768]
Re: https://forums.ankiweb.net/t/anki-2-1-50-beta-6-release-candidate/18181/92

This can be reproduced on a Mac as well, and only seems to happen in an optimized qt5 build:

```
BUILDARGS="-c opt" ./tools-run-qt5.14
```

Renaming the tag name on this line:

    .autocomplete-reference {

seems to allow the text to appear, so it seems to be somewhere in that area. The top/left clauses are not being included in the exported CSS in a release build:

.bazel/bin/ts/editor/browser_editor.css:

```
autocomplete-reference.svelte-1a0b51q{position:absolute\
;inset:0;border-left:1px solid transparent;border-top:2px solid transparent}
```

Same output from a qt6 build (unsurprisingly), but the newer web toolkit seems to handle it differently.

Would this be an easy fix for 2.1.50?

**Comment by hikaru-y** [comment]
> The top/left clauses are not being included in the exported CSS in a release build:

If the `minify` attribute is set to `True` and the `target` attribute to its default value (`es2015`) in the esbuild rules, it appears that `top`, `right`, `bottom`, and `left` are converted to `inset`, which is the shorthand for them. The chromium shipped with Qt 5.14 or 5.15 is old and does not seem to support `inset`.
https://caniuse.com/mdn-css_properties_inset
https://wiki.qt.io/QtWebEngine/ChromiumVersions

I made the following modifications to `ts/esbuild.bzl` and ran Qt5 opt build on Windows 10 with the following command, and then confirmed that `top`, `right`, etc. are not converted to `inset` and the tag input works fine.
```diff
diff --git a/ts/esbuild.bzl b/ts/esbuild.bzl
index 6eb45defc..332e9da18 100644
--- a/ts/esbuild.bzl
+++ b/ts/esbuild.bzl
@@ -17,5 +17,7 @@ def esbuild(name, **kwargs):
             "//:release": True,
             "//conditions:default": False,
         }),
+        # support Qt 5.14
+        target = "chrome77",
         **kwargs
     )
```
```shell
PS C:\anki> $env:BUILDARGS="-c opt"
PS C:\anki> .\tools\run-qt5.15.bat
```

**Comment by dae** [comment]
Thanks for investigating this and coming up with a fix, Hikaru! :tada:
